### PR TITLE
Update to OpenVR 1.0.11, Support Multiple Vive Trackers

### PR DESCRIPTION
**Pull request by gfrolov**
This pull request updates the OSVR-Vive to the latest OpenVR 1.0.11 and also adds functionality to support multiple Vive trackers - #36

### CODE REVIEW

File: PropertyTraits.h [hunk 1]
```diff
@@ -145,6 +145,8 @@ namespace vive {
             vr::Prop_DriverDirectModeSendsVsyncEvents_Bool,
         DisplayDebugMode = vr::Prop_DisplayDebugMode_Bool,
         GraphicsAdapterLuid = vr::Prop_GraphicsAdapterLuid_Uint64,
+        DriverProvidedChaperonePath =
+            vr::Prop_DriverProvidedChaperonePath_String,
```
Comment (rpavlik) [inline PropertyTraits.h]: Was this updated with the tool or manually? (Just wanted to make sure it was the tool so that it's complete)

**Comment by rpavlik** [comment]
I suspect you left some commits out - I only see 1 commit that mostly looks like an OpenVR 1.10 update

File: PropertyTraits.h (hunk 1 above)
Comment (mars979) [reply PropertyTraits.h]: Yup, there were not a lot of changes there

**Comment by DuFF14** [comment]
Tested that multiple puck tracking is working fine when I build RelWithDebInfo. Server crashes when I start the server with a Release build of the plugin. Could be a dependency issue on my end?

**Comment by mars979** [comment]
The CMake is most likely linking against debug libraries if it cannot find Release type libraries. Since OSVR-Core is built with RelWithDebInfo, the CMake will choose debug instead.
I built the project in Release mode and confirmed that's the case for me as well.

**Comment by DuFF14** [comment]
Yea, it was linking against the debug libraries. Release build works if I edit that manually. Code looks fine to me.

**Pull request merged**